core/templates/core/email/fingerprint.html

```diff
@@ -0,0 +1,3 @@
+Location: {{ location }}
```

Comment:
This is included in most emails that get sent out. I like using templates solely, much better than just tacking this onto an email body.